config/credentials.yml.enc

```diff
@@ -0,0 +1 @@
+NXoqOVwhh8kaZGlkYeinVH9aK1n8T8aCXFgcMaVnbwYmvzZaDe8tvJ/7nRd6JdvwKVZIlkZyPzeEAL3AEwfXXyX8xrahj+AOhGLNH5md+Pg4ICtDENLKikvpgVjJSN7ovsu7STR+jvfcZor/9xU5Sbt5kfLMYimxa9SncvJwkQyssFsG+UppHT5fWSbzDcsCXJypg9yO/sxNKz91MmU04nFF48HcHPdWjKwPaVXvSOQ2VIDTEe3E+Yxnm/73Y3kiF1qCeP5Cr4xmyz6VkU6DXutSdmBrpXr2XO1PrhQlTMPwN9mnJqxJsYuhjQ7honmHc1aQ3rdLXC/Jv3x2AX6NL5Eii3wy9V7Kb/yDOubZpQNUzjMxNuWapn75v6SM+c2K1AmyocXdoztsg7DviSio7I0dIzRkfa42jDly--ifjSeUrS3r/F5ncq--bUclqVfJrNWCOW5mOKWMlA==
```

Comment:
Hadn't seen this before. Can confirm [this should be added to version control](https://blog.saeloun.com/2019/10/10/rails-6-adds-support-for-multi-environment-credentials.html#:~:text=key-,config%2Fcredentials.,to%20our%20version%20control%20systems.).